applications/FemToDemApplication/custom_python/fem_to_dem_python_application.cpp

```diff
@@ -61,6 +61,8 @@ namespace Python
 	AddCustomProcessesToPython(m);
 
 	//registering variables in python
+	//KRATOS_REGISTER_IN_PYTHON_VARIABLE(m,DEM_PARTICLE_POINTER);

```

Comment:
Remove if unused

Reply:
ok, it is used but not registered in python due to compilation problems